scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPrediction.scala

```diff
@@ -51,6 +52,7 @@ trait H2OMOJOPrediction
   }
 
   def getPredictionUDF(): UserDefinedFunction = {
+    SQLConf.get.setConfString("spark.sql.legacy.allowUntypedScalaUDF", "true")
```

Comment:
I'm fine with this for now. Just qq: Is there a change that future versions will remove support for this, what the alternative would be then?

Reply:
In future versions, we could use `def udf(f: UDF2[_, _, _], returnType: DataType): UserDefinedFunction` but is available since Spark 2.3 or create `SparkUserDefinedFunction` function directly (in spark 2.1 is named UserDefinedFunction).

I will implement some workaround for Spark 3.0 so that the option is not needed

Reply:
Thanks for the info!